src/seedu/addressbook/parser/Parser.java

```diff
@@ -408,18 +421,23 @@ private Command prepareViewAll(String args) {
      * @throws NumberFormatException the args string region is not a valid number
      */
     private int parseArgsAsDisplayedIndex(String args) throws ParseException, NumberFormatException {
-        final Matcher matcher = PERSON_INDEX_ARGS_FORMAT.matcher(args.trim());
+        final Matcher matcher = GENERAL_ARGS_FORMAT.matcher(args.trim());
         if (!matcher.matches()) {
             throw new ParseException("Could not find index number to parse");
         }
-        return parseInt(matcher.group("targetIndex"));
+        return parseInt(matcher.group("keywords"));
     }
 
     /**
      * Parse the given arguments string as a float
 
+<<<<<<< HEAD
+    private float parseArgsAsFloat(String args) throws ParseException, NumberFormatException {
+        final Matcher matcher = GENERAL_ARGS_FORMAT.matcher(args.trim());
+=======
      private float parseArgsAsFloat(String args) throws ParseException, NumberFormatException {
         final Matcher matcher = PERSON_INDEX_ARGS_FORMAT.matcher(args.trim());
+>>>>>>> upstream/master
         if (!matcher.matches()) {
```

Comment:
Some merge issues here

Reply:
oh i thought somebody was gonna use that function since its all commented out

Reply:
This appeared during your merge, so it is within responsiblity to ensure the merge terminates properly, rather than leaving unknown code fragments in the code base. If you are unsure as to how to merge it appropriately, the action to take is to ask the writer of the code, ie who you merged from

Reply:
And also this

Reply:
I was wondering why Github didn't tell me it's outdated, then I realized I commented on a different line lawl. My bad